src/components/elements/inputs/boolean-list.js

```diff
@@ -9,8 +9,16 @@ const isChecked = (values, option) => {
   }
 }
 
-export default class CheckboxList extends Input {
+export default class BooleanList extends Input {
+  isMultiSelect(){
```

Comment:
I would make this a property instead (and remove "select" since it's not a select):

```javascript
get isMulti() {
  return this.props.property.type === 'string[]'
}
```